src/Symfony/Component/Runtime/StartedAppInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Runtime;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface StartedAppInterface
```

Comment:
App ->Application
I would also use application everywhere instead of app (including in the docs)